inference-engine/tests/functional/shared_test_classes/src/single_layer/experimental_detectron_detection_output.cpp

```diff
@@ -0,0 +1,154 @@
+        attributes.num_classes,
+        attributes.post_nms_count,
+        attributes.max_detections_per_image,
+        attributes.class_agnostic_box_regression,
+        attributes.deltas_weights,
+        netPrecision,
+        targetName) = obj.param;
+
+    std::ostringstream result;
+
+    using ov::test::operator<<;
+    result << "input_rois=" << inputShapes[0] << "_";
+    result << "input_deltas=" << inputShapes[1] << "_";
+    result << "input_scores=" << inputShapes[2] << "_";
+    result << "input_im_info=" << inputShapes[3] << "_";
+
+    using ov::test::subgraph::operator<<;
+    result << "attributes={" << attributes << "}_";
+    result << "netPRC=" << netPrecision << "_";
+    result << "trgDev=" << targetName;
+    return result.str();
+}
+
+void ExperimentalDetectronDetectionOutputLayerTest::SetUp() {
+    std::vector<InputShape> inputShapes;
+    ngraph::opset6::ExperimentalDetectronDetectionOutput::Attributes attributes;
+
+    ElementType netPrecision;
+    std::string targetName;
+    std::tie(
+        inputShapes,
+        attributes.score_threshold,
+        attributes.nms_threshold,
+        attributes.max_delta_log_wh,
+        attributes.num_classes,
+        attributes.post_nms_count,
+        attributes.max_detections_per_image,
+        attributes.class_agnostic_box_regression,
+        attributes.deltas_weights,
+        netPrecision,
+        targetName) = this->GetParam();
+
+    inType = outType = netPrecision;
+    targetDevice = targetName;
+
+    init_input_shapes(inputShapes);
+
+    auto params = ngraph::builder::makeDynamicParams(netPrecision, {inputDynamicShapes});
+    auto paramsOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+    auto experimentalDetectron = std::make_shared<ngraph::opset6::ExperimentalDetectronDetectionOutput>(
+        params[0], // input_rois
+        params[1], // input_deltas
+        params[2], // input_scores
+        params[3], // input_im_info
+        attributes);
+    function = std::make_shared<ov::Function>(
+        ov::OutputVector{experimentalDetectron->output(0), experimentalDetectron->output(1)},
+        "ExperimentalDetectronDetectionOutput");
+}
+
+namespace {
+template <class T>
+ov::runtime::Tensor createTensor(
+        const ov::element::Type& element_type,
+        const Shape& shape,
+        const std::vector<T>& values,
+        const size_t size = 0) {
+    const size_t real_size = size ? size : values.size() * sizeof(T) / element_type.size();
+    ov::runtime::Tensor tensor { element_type, shape };
+    std::memcpy(tensor.data(), values.data(), std::min(real_size * element_type.size(), sizeof(T) * values.size()));
+    return tensor;
+}
+} // namespace
+
+void ExperimentalDetectronDetectionOutputLayerTest::generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticShapes) {
+    std::vector<ov::runtime::Tensor> inputTensors = {
+        // 16 x 4 = 64
```

Comment:
what if `targetInputStaticShapes` will be contain shape different from expected?

Reply:
done, thanks